### ISSUE TITLE
Missing transition utils

### ISSUE THREAD

**Issue by leoeuclids** [miguelcobain/ember-css-transitions#120]
## Description

Importing functions from transition utils, like described in the [docs](https://miguelcobain.github.io/ember-css-transitions/docs/api/modules/ember-css-transitions/utils/transition-utils), doesn't work after version 4.0.0.
I'm not sure if this is intended, but if someone can confirm this should still be working, I can open a PR to fix it.

## How to reproduce

1. Add `import { nextTick } from 'ember-css-transitions/utils/transition-utils';`
2. Run `ember s`
3. Should result in `webpack returned errors to ember-auto-import`

## Source

The utils folder is not being defined as a [public entry point](https://github.com/miguelcobain/ember-css-transitions/blob/master/ember-css-transitions/rollup.config.mjs#L17) in the rollup config file.

**Comment by miguelcobain** [comment]
I suspect the change to v2 addon format.

@SergeAstapov do you know why this happens?

**Comment by tcjr** [comment]
I'm seeing this in a project where I'm updating dependencies.  It looks like the utils aren't in the NPM package.

https://www.npmjs.com/package/ember-css-transitions?activeTab=code

**Comment by SergeAstapov** [comment]
> I suspect the change to v2 addon format.
> 
> @SergeAstapov do you know why this happens?

ah yes, we didn't list `/utils` as public entry point hence it's not available.
Thanks @tcjr for the PR!

**Comment by matthewhartstonge** [comment]
👋 just hit this with `v4.4.0`.
Are we able to get a release including the bug fix above? 🙏

**Comment by matthewhartstonge** [comment]
Worked out that in `v4.4` these utils have migrated from:

```js
import { nextTick } from 'ember-css-transitions/utils/transition-utils';
```

To:
```js
import { nextTick } from 'ember-css-transitions/modifiers/css-transition';
```

**Comment by matthewhartstonge** [comment]
So those functions, even though they are being inlined into `/modifiers/css-transition`, aren't exported.

This is definitely a bug.

The documentation also mentions that we should be able to [pull in the utils](https://miguelcobain.github.io/ember-css-transitions/docs/api/modules/ember-css-transitions/utils/transition-utils).

@miguelcobain or @SergeAstapov are we able to get #138 merged and released?

**Comment by SergeAstapov** [comment]
The fix now released as v4.4.1
